keyboards/cutie_club/keebcats/gina/info.json

```
@@ -0,0 +1,85 @@
+{
+    "keyboard_name": "Keebcats Gina",
+    "manufacturer": "Cutie Club",
+    "maintainer": "Cutie Club",
+    "url": "",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true
```

migrated from `rules.mk`
```suggestion
        "extrakey": true,
        "rgblight": true
```